cuda.spec

```diff
@@ -60,9 +63,21 @@ rm -rf %_builddir/bin/nsight
 #ln -sf ../libnvvp/nvvp %_builddir/bin/nvvp
 rm -rf %_builddir/bin/nvvp
 rm -rf %_builddir/bin/computeprof
+
+# add a wrapper for cuda-gdb and package the support files
+mkdir %{i}/share
+cp -ar %_builddir/share/gdb/ %{i}/share/
+mv %_builddir/bin/cuda-gdb %_builddir/bin/cuda-gdb.real
+cat > %_builddir/bin/cuda-gdb << @EOF
+#! /bin/bash
+export PYTHONHOME=\$(scram tool tag python PYTHON_BASE)
```

Comment:
@fwyzard , you can add python dependency and use $PYTHON_ROOT in the wrapper. Do not forget to relocate the wrapper in %post

Reply:
Mhm, I'd rather not add to cuda the dependency on python, to avoid rebuilding cuda too often.
I'll stick with finding it at runtime, and maybe change it later.

Reply:
@fwyzard , ok valid point. How about creating a new cuda-gdb.spec  which depends on both cuda and python? You will also need cuda-gdb-toolfile.spec .

Reply:
I thought about it, but the hour was getting late...

What happens if both `cuda` and `cuda-gdb-wrapper` have a binary called `cuda-gdb` ?

Reply:
@fwyzard , if there is no dependency in between cuda-gdb-wrapper and cuda toolfiles then itis undefined behaviour. But if you make cuda-gdb-wrapper toolfile depend on cuda then cuda-gdb from cuda-gdb-wrapper will be in PATH via exteranls/arch/bin directory and cuda-gdb from cuda will end up in externals/arch/bin00 direcotry (which is not in PATH).

Reply:
Ah, so that's what `.../bin00` is about...

Anyway, I've kept the renaming in `cuda.spec` and moved the rest to a separate spec and tool file, so it should be OK now.